test/evergreen.yml

```diff
@@ -3476,6 +3476,43 @@ tasks:
           times: 1       # No of times to run the loop
           no_of_procs: 10 # No of processes to run in the background
 
+  - name: skiplist-stress-test
+    tags: ["stress-test-1", "stress-test-zseries-1"]
+    exec_timeout_secs: 3600 # 1 hour
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v4_gcc.cmake
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/bench/workgen/runner"
+          script: |
+            set -o errexit
+            set -o verbose
+            for i in $(seq 150); do
```

Comment:
Have we tested we can successfully run this many iterations without it timing out within the hour? 

As an additional test, we would not want these tests to time out on ordinary runs.

Reply:
Thanks, good call - 150 was much too high. I've done some testing and ~5 was the highest that would reliably run, so I've pushed that in 99b4dc5.